src/include/mpir_handlemem.h

```diff
@@ -233,6 +233,9 @@ Input Parameters:
   +*/
 static inline void *MPIR_Handle_obj_alloc(MPIR_Object_alloc_t * objmem)
 {
+    /* MPI_Info must call MPIR_Handle_obj_alloc_noinit(). */
```

Comment:
`MPIR_Handle_obj_alloc_noinit` has renamed, right?

Reply:
Yes, you're right. I changed it to `MPIR_Info_handle_obj_alloc()`.
I also fixed other places that use `MPIR_Info_handle_obj_alloc()`. Now `_noinit` does not appear in this PR.